src/validators/relatives.js

```diff
@@ -18,11 +27,40 @@ export default class RelativesValidator {
     })
   }
 
+  validMaritalRelations(context = null) {
+    const maritalStatuses = ['Married', 'Separated']
```

Comment:
Just noting that while this is consistent with validation matrix, because we have been treating Married as a different thing than InCivilUnion, this would be a defect. However now that we are coalescing Married/InCivilUnion in #918, it is moot.

Reply:
Resolved with https://github.com/18F/e-QIP-prototype/pull/927